code/modules/cargo/packs.dm

```diff
@@ -609,6 +663,7 @@
 	
 /datum/supply_pack/engineering/engine
 	name = "Emitter Crate"
+	desc = "Useful for powering forcefield generators while destroying locked crates and intruders alike. Contains two high-powered energy emitters."
```

Comment:
This one's missing the "requires CE access" line.
Also holy shit, props for writing all these descriptions.